src/Tribe/RSVP.php

```diff
@@ -926,6 +926,10 @@ public function save_ticket( $event_id, $ticket, $raw_data = array() ) {
 			return false;
 		}
 
+		// Updates if we should show Description
+		update_post_meta( $ticket->ID, tribe( 'tickets.handler' )->key_show_description, $ticket->show_description );
```

Comment:
same concern here as in https://github.com/moderntribe/event-tickets-plus/pull/436